src/tests/suite/daos_container.c

```diff
@@ -95,14 +95,14 @@ co_attribute(void **state)
 	daos_event_t	 ev;
 	int		 rc;
 
-	char const *const names[] = { "AVeryLongName", "Name" };
+	char const *const names[] = { strdup("AVeryLongName"), strdup("Name") };
```

Comment:
jvolivie-desk1:[~]$ ./a.out
Trying foo[1] = 'y', foo = HelloWorld
foo = HylloWorld
Trying bar[1] = 'y', bar = HelloWorld

Segmentation fault (core dumped)


The difference between foo and bar

	char foo[] = {"HelloWorld"};
	const char *bar = {"HelloWorld\n"};

Reply:
yeah and then you shouldn't need a strdup